application.md

```diff
@@ -1,5 +1,141 @@
+source development of astronomy software. 
+We are submitting this application together in order to strengthen collaboration
+between the different projects, for example by offering project ideas that
+extend across multiple packages, in order to reduce duplication of efforts and
+strengthen interoperability.
+As an umbrella organization with an astronomy theme, we will be able to include
+diverse projects (not only Python-based) and accept students with experience
+with different languages, while uniting projects with the common goal of
+providing tools for the scientific exploration of the Universe.
+
+By participating in GSoC as an umbrella, OpenAstronomy aims to improve the
+experience for both students and mentors alike, while reducing the
+administrative overhead.
+Both the SunPy and Astropy projects have participated in GSoC from 2013 to 2015
+under the Python Software Foundation umbrella organization, and the experience
+was highly successful for both organisations (a total of X students completed
+their projects during these years, many of them with exceptional performance).
+Additionally the yt project participated last year under NumFocus with positive
+results. 
+We hope that we can replicate and share this experience with other projects as
+part of the OpenAstronomy organization.
+
+## How many potential mentors have agreed to mentor this year?
+
+At the moment we have XX potential mentors for this year’s program (listed on
+our ideas page).
+In order to be selected as mentors, developers need to have made major code
+contributions to their respective project, and be active contributors to date.
+These are therefore 'core' developers that understand the code base very well.
+They must also be in a position to dedicate enough time to properly mentor GSoC
+students.
+A number of the mentors have been GSoC mentors in all the editions since 2013.
+How will you keep mentors engaged with their students? Each sub-organisation has
+weekly teleconferences to discuss theIr development, in these calls they will
+dedicate a time for the GSoC mentors and students to update the progress and
+problems encountered.
+
+/is that engaging enough? A Party or chocolates will have more effect,
+something that will make others to want to mentor in the future/
+
+## How will you help your students stay on schedule to complete their projects?
+
+The students are asked to keep a blog up to date with their progress.
+Additionally, from the timeline in their application and with the help of the
+mentors a number of milestones will be set.
+All this information will be publicly available.
+In case of problems or delays most of the mentors have experience in help the
+students to identify the causes to drag them behind.
+The organisation admins will keep a close eye on the projects with inexperienced
+mentors to be able to help before to late.
+If after all the efforts the student don't show interest or disappears, the
```

Comment:
I think this final sentence is unnecessary and sets a negative tone implying we expect the students to fail. Instead I would say something a bit more specific like "The admins will also monitor the progress of all the students based on weekly updates from the mentors.  If despite intervention they find the students have fallen behind on multiple milestones without specific allowable reasons, they will be warned of possible failure, in the hopes that this will put them more on track.  If this does not succeed, they will be failed."